README.md

```diff
@@ -23,15 +23,17 @@ For more information, see the blog post [Nextcloud in Digital Imaging](https://n
 
 ### Installation
 
-On your Nextcloud, simply navigate to Apps > Multimedia > DICOM Viewer, and enable it.
+Build the app using the instructions below and then on Nextcloud, simply navigate to Apps > Multimedia > DICOM Viewer, and enable it.
```

Comment:
No need to build for end users to use the released version. It is available under Apps > Multimedia on Nextcloud.